docker-compose.yml

```diff
@@ -1,53 +1,102 @@
 version: '3.9'
 
 services:
-  dm-record-manager:
-    image: 'ofn-record-manager:latest'
-    container_name: dm-record-manager
+  record-manager-ui:
+    image: 'ghcr.io/kbss-cvut/kbss-cvut/record-manager-ui:latest'
     ports:
-      - '4000:80'
+      - "127.0.0.1:3000:80"
     depends_on:
-      - dm-record-manager-server
+      - record-manager
     environment:
-      API_URL: "http://localhost:3000/ofn-record-manager"
+      APP_TITLE: "Record Manager"
+      BASENAME: "./"
+      LANGUAGE: "cs"
+      NAVIGATOR_LANGUAGE: "true"
+      API_URL: "http://localhost:8080/record-manager"
```

Comment:
```suggestion
      API_URL: "http://localhost:8080/record-manager-server"
```

Reply:
Maybe this requires also "context-path" setting in  service "record-manager-server".